### DISCUSSION TITLE
Allow TARDIS's wall blocks and light blocks (display entity-based ones) to be crafted in survival.

### DISCUSSION THREAD

**Discussion by MagmaSlime123** [eccentricdevotion/TARDIS#712] [Ideas]
Title. Since wall blocks and lights now use display entities, it'd be nice if place-able survival-obtainable ones were craft-able, since both creative and survival users cannot replace them if broken by creative users.

Also, wall blocks seem to be unbreakable in survival, and the /tgive <Username> mushroom 1 command says the item doesn't exist.

**Comment by eccentricdevotion** [comment]
Implemented crafting
`/tgive mushroom 1` is not a valid command - mushroom blocks are no longer used in the plugin
